dev/src/templates/JavaSpring401/libraries/spring-boot/openapi2SpringBoot.mustache

```diff
@@ -0,0 +1,178 @@
+
+import org.openapitools.jackson.nullable.JsonNullableModule;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ExitCodeGenerator;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+{{^reactive}}
+import org.springframework.format.FormatterRegistry;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.validation.MessageCodesResolver;
+import org.springframework.validation.Validator;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.HandlerMethodReturnValueHandler;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.config.annotation.AsyncSupportConfigurer;
+import org.springframework.web.servlet.config.annotation.ContentNegotiationConfigurer;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+    {{^useSpringfox}}
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+    {{/useSpringfox}}
+import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewResolverRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+    {{^java8}}
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+    {{/java8}}
+{{/reactive}}
+{{#reactive}}
+import org.springframework.web.reactive.config.CorsRegistry;
+    {{^useSpringfox}}
+import org.springframework.web.reactive.config.ResourceHandlerRegistry;
+    {{/useSpringfox}}
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+{{/reactive}}
+
+@SpringBootApplication
+@ComponentScan(basePackages = {"{{basePackage}}", "{{apiPackage}}" , "{{configPackage}}"})
+public class OpenAPI2SpringBoot implements CommandLineRunner {
+
+    @Override
+    public void run(String... arg0) throws Exception {
+        if (arg0.length > 0 && arg0[0].equals("exitcode")) {
+            throw new ExitException();
+        }
+    }
+
+{{#mavenPomExists}}
+//    Uncomment and explicitly configure this main class.  For example, provide the <configuration><mainClass> or <start-class> elements in the pom.xml
+//    Alternatively, merge the contents of this class, including the above @ComponenScan packages, with the existing implicit or explicit main class.
```

Comment:
We cannot use "above" because of accessibility reasons. Instead, say something like, "including the previous..." or "including the previously mentioned..."

Reply:
Ok

Reply:
Here are the updates:

//    The OpenAPI Tools disabled this main class so that the existing implicit or explicitly configured main class is not overridden.
//    To use this main class and enable the OpenAPI endpoints, uncomment the following lines, and explicitly configure this main class.
//    For example, provide the <configuration><mainClass> or <start-class> elements in the pom.xml file. Alternatively, merge the contents
//    of this class, including the packages listed in the @ComponenScan annotation, with the existing main class.

Reply:
For the first line, it should probably be "implicitly or explicitly configured..." to be parallel. Thanks for the changes!

Reply:
Ok

Reply:
@sishida , all the updates have been added to this PR